src/game/chars/CCharSkill.cpp

```diff
@@ -2459,6 +2459,27 @@ int CChar::Skill_Herding( SKTRIG_TYPE stage )
 	{
 		case SKTRIG_START:
 		{
+			// CanSeeLos if pChar or m_Act_P
+			if ( !CanSeeLOS(pChar) || !CanSeeLOS(m_Act_p) )
+			{
+				SysMessageDefault(DEFMSG_MSG_MOUNT_DIST);
+				return -SKTRIG_ABORT;
+			}
+
+			// tamed pets cannot be herded
+			if ( !pChar->IsStatFlag(STATF_PET) )
+			{
+				SysMessagef(g_Cfg.GetDefaultMsg( DEFMSG_TAMING_TAME ), pChar->GetName());
+				return -SKTRIG_ABORT;
+			}
+
+			// is a valid point?
+			if ( !m_Act_p.IsValidPoint())
+			{
+				SysMessage("That location is not valid.");
```

Comment:
Is there a defmsg also for this?

Reply:
Missed, added new DEFMSG_LOCATION_INVALID for invalid locations